src/types/constants.go

```diff
@@ -41,4 +41,10 @@ const (
 	// K8sNetworkPolicy
 	K8sNwPolicyAPIVersion = "networking.k8s.io/v1"
 	K8sNwPolicyKind       = "NetworkPolicy"
+
+	// Operation Types
+	OperationTypeFile    = "File"
```

Comment:
```suggestion
	SumOpTypeFile    = "File"
```

Reply:
Thanks for the suggestions! 
Suggestions are updated.